src/modules/Api.ts

```diff
@@ -65,22 +72,41 @@ export class Api {
     expiration: Integer | string,
     fillOrKill: boolean,
     postOnly: boolean,
+    triggerPrice?: Integer,
+    signedTriggerPrice?: Integer,
+    decreaseOnly?: boolean,
     clientId?: string,
   }): Promise<{ order: ApiOrder }> {
-    const order: SignedLimitOrder = await this.createOrder({
-      makerAccountOwner,
-      makerMarket,
-      takerMarket,
-      makerAmount,
-      takerAmount,
-      makerAccountNumber,
-      expiration,
-    });
+    let order: SignedLimitOrder | SignedStopLimitOrder;
+    if (signedTriggerPrice || decreaseOnly) {
```

Comment:
we can just do `if (triggerPrice)` and do everything off of that one variable rather than checking any one of three or something like that